resources/commit_messages.md

```diff
@@ -0,0 +1,13 @@
+Commit Messages
+=======
```

Comment:
A valid title in markdown should contain the size of characteres for `=`, for example:

```md
Commit Messages
===============
```

Or

```
# Commit Messages
```

Reply:
Fixed in 83f2a53